module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp

```diff
@@ -165,19 +150,7 @@ namespace module::behaviour::planning {
                 }
 
                 if (latestCommand.type == message::behaviour::MotionCommand::Type::STAND_STILL) {
-
-
                     emit(std::make_unique<StopCommand>(subsumptionId));
-                    // emit(std::make_unique<ActionPriorities>(ActionPriorities { subsumptionId, { 40, 11 }}));
-
-                    return;
-                }
-                else if (latestCommand.type == message::behaviour::MotionCommand::Type::DIRECT_COMMAND) {
-                    // TO DO, change to Bezier stuff
-                    std::unique_ptr<WalkCommand> command =
-                        std::make_unique<WalkCommand>(subsumptionId, latestCommand.walk_command);
-                    emit(std::move(command));
-                    emit(std::make_unique<ActionPriorities>(ActionPriorities{subsumptionId, {40, 11}}));
```

Comment:
Hesitant to say removing this is "fine" maybe @TrentHouliston or @Bidski should have a look

Reply:
This shouldn't have been removed.